mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipperTest.java

```diff
@@ -18,35 +18,28 @@
  ****************************************************************/
 package org.apache.james.mailbox.backup;
 
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MAILBOX_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MAILBOX_1_SUB_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MAILBOX_2;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MAILBOX_ID_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_2;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_CONTENT_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_CONTENT_2;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_ID_1;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_ID_2;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.MESSAGE_UID_1_VALUE;
-import static org.apache.james.mailbox.backup.MailboxMessageFixture.SIZE_1;
+import static org.apache.james.mailbox.backup.MailboxMessageFixture.*;
```

Comment:
BTW It's probably checked by our checkstyle rules. Maybe you can ask a colocated mate how to configure checkstyle in your IDE.

Reply:
@aduprat has just given me it's checkstyle file :)

Reply:
the official one is at the root of james-project